controller/readonly.go

```diff
@@ -29,11 +30,6 @@ type ReadOnly struct {
 
 // NewReadOnly configures and initializes a new ReadOnly controller
 func NewReadOnly(conf *config.Config) (*ReadOnly, error) {
-	// Run the driver with logging to output the Terraform plan to stdout
-	if tfConfig := conf.Driver.Terraform; tfConfig != nil && !MuteReadOnlyController {
-		tfConfig.Log = config.Bool(true)
```

Comment:
Good question! Thanks for asking. You're right, it was no longer used. For example if you go to an earlier commit (before all this refactoring) and hardcode this to false, the plan still outputs to the terminal. This is because we switched to reading the plan into a buffer and logging it in the [old checkInspect code](https://github.com/hashicorp/consul-terraform-sync/pull/808/files#diff-8e409a7329d9bad51e5c0d25844fa2d15602caab402947e2d4e8b60951e93e3bL328-L332) I removed. `MuteReadOnlyController` is so far only used by our benchmarks but I haven't checked how those are outputting.

That makes me realize I'm not outputting the plan URL like the old inspect code! I'll add a commit to make this the same